### ISSUE TITLE
Change color of extension icon if current tab already snoozed?

### ISSUE THREAD

**Issue by orschiro** [rohanb10/snoozz-tab-snoozing#20]
Hi @rohanb10, 

This would be a great feature to prevent duplicated snoozes!

Thoughts?

**Comment by rohanb10** [comment]
Yes this is a good idea, but I don't want to add this in until I add the ability to edit wake up times. I feel like these two go hand in hand, instead of snoozing again and deleting the old one

**Comment by orschiro** [comment]
That makes sense, thanks!

**Comment by rohanb10** [comment]
Thought about this for a while and I'm going to have to backtrack a bit. I won't be implementing this change.

Basically in order to have the icon change when you're on a page you have snoozed before, I would have to go through your list of sleeping tabs every single time you switch a tab. This will probably be fine if you only have like 10-15 things in your history, but I know of several users with 100+ items in their history. This would be quite a large background task to execute every single time you switch a tab, an action that is performed 100s of times a day.

I still think the idea is sound, so I might think about making a banner or some sort of visual cue in the popup itself. That way it will only run when the popup opens and wont have a huge performance overhead in the background.

This is pretty low priority and right now my focus is on adjusting the UI to allow recurring or repeat snoozes.
